### PR TITLE
Build zbchaos docker image

### PR DESCRIPTION
**Pull request by oleschoenburg**
Depends on #191 

Builds a docker image `gcr.io/zeebe-io/zbchaos` that runs zbchaos (with the worker command by default).

### CODE REVIEW

File: go-chaos/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# syntax=docker/dockerfile:1.4
+FROM golang:alpine as builder
+
+WORKDIR /app 
+
+COPY --link go.mod go.sum .
+RUN go mod download
+
+COPY --link cmd/ ./cmd
```
Comment (Zelldon) [inline go-chaos/Dockerfile:9]: TIL: --link creates a symlink? Wasnt aware of this feature :+1:

**Review by Zelldon — APPROVED** [review]
Mega :rocket:

File: go-chaos/Dockerfile (hunk 1 above)
Comment (oleschoenburg) [reply go-chaos/Dockerfile:9]: Not exactly but it allows to copy files independently from previous layers: https://docs.docker.com/engine/reference/builder/#copy---link

Comment (Zelldon) [reply go-chaos/Dockerfile:9]: Ok this is not what I expected but OK :D

Comment (oleschoenburg) [reply go-chaos/Dockerfile:9]: Yeah, I feel like a lot of the more recent docker features are really subtle and not easy to understand :grimacing:

**Pull request merged**